bin/stacks/routing-lambda-stack.ts

```diff
@@ -53,6 +58,21 @@ export class RoutingLambdaStack extends cdk.NestedStack {
 
     const region = cdk.Stack.of(this).region
 
+    const env = {
+      VERSION: '2',
+      JSON_RPC_PROVIDER: jsonRpcProvider,
+      NODE_OPTIONS: '--enable-source-maps',
+      POOL_CACHE_BUCKET: poolCacheBucket.bucketName,
+      POOL_CACHE_BUCKET_2: poolCacheBucket2.bucketName,
+      POOL_CACHE_KEY: poolCacheKey,
+      TOKEN_LIST_CACHE_BUCKET: tokenListCacheBucket.bucketName,
+      ETH_GAS_STATION_INFO_URL: ethGasStationInfoUrl,
+    }
+
+    jsonRpcProviderOverride.forEach((url: string, chainId: ChainId) => {
+      ;(env as any)[`JSON_RPC_PROVIDER_${ID_TO_NETWORK_NAME(chainId).toUpperCase}`] = url
+    })
```

Comment:
I don't see this used? I think its best to just pass in the ID anyway, then compute the url for each chain